### PR TITLE
F/cart delete

### PR DESCRIPTION
**Pull request by emkquinn**
- [x] added delete button to items in cart
- [x] updates cart appropriately when deleting item for logged in user and non logged in user

### CODE REVIEW

File: client/store/cart.js [hunk 1]
```diff
@@ -101,6 +103,28 @@ export const setCart = userId => async dispatch => {
...
 }
 
+export const removeFromCart = (item, orderId, userId) => async dispatch => {
+  try {
+    if (userId) {
+      await axios.delete(`api/users/${userId}/cart`, {data: {item, orderId}})
+    } else {
+      const localCart = JSON.parse(window.localStorage.getItem('cart'))
+      const products = localCart.products
+      const updatedProducts = products.filter(product => {
```
Comment (brynn) [inline client/store/cart.js]: aaaaamazing!

File: client/store/cart.js [hunk 2]
```diff
@@ -101,6 +103,28 @@ export const setCart = userId => async dispatch => {
...
+        if (product.id !== item.id) {
+          return product
+        }
+      })
+      console.log('updatedProducts', updatedProducts)
```
Comment (brynn) [inline client/store/cart.js]: remove por favor 😜

File: client/utils/index.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import _ from 'lodash'
 import ProductCard from '../components/ProductCard'
+import {compose} from 'redux'
```
Comment (brynn) [inline client/utils/index.js]: are you using this anywhere?

File: client/store/cart.js (hunk 2 above)
Comment (emkquinn) [reply client/store/cart.js]: lol I always forget the console.logs!

File: client/utils/index.js (hunk 3 above)
Comment (emkquinn) [reply client/utils/index.js]: No...not sure what that's from. Maybe it was imported automatically through a typo

**Pull request merged**